common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -5,11 +5,9 @@ define('video/04_video_full_screen.js', [], function () {
         '<button class="control add-fullscreen" aria-disabled="false" title="',
             gettext('Fill browser'),
         '">',
-            '<span class="icon-fallback-img">',
-                '<span class="icon fa fa-arrows-alt" aria-hidden="true"></span>',
-                '<span class="sr control-text">',
-                    gettext('Fill browser'),
-                '</span>',
+            '<span class="icon fa fa-arrows-alt" aria-hidden="true"></span>',
```

Comment:
Hi @alisan617, I was watching your changes and view a couple of issues, maybe you should keep them in mind.

You removed `.icon-fallback-img` container, you left the class `.icon`, but this class is nested, for this reason the class is useless and the icon element will lose this properties:
``` CSS
.icon-fallback-img .icon {
     width: 1em;
     height: 1em;
}
```
This same happens in other files.